overlord/configstate/configcore/homedirs_test.go

```diff
@@ -264,6 +220,11 @@ func (s *homedirsSuite) TestConfigureHomedirsHappy(c *C) {
 	})
 	defer restore()
 
+	restore = configcore.MockApparmorSetupSnapConfineSnippets(func() (bool, error) {
+		return false, nil
```

Comment:
don't we also need to check that this gets called as needed?

Reply:
Good point, thanks!